samples/crypto/sha256/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -7,3 +7,5 @@ CONFIG_TFM_PROFILE_TYPE_NOT_SET=y
 
 # Using hardware crypto accelerator
 CONFIG_PSA_CRYPTO_DRIVER_CC3XX=y
+
+CONFIG_MBEDTLS_ENABLE_HEAP=n
```

Comment:
Not related only to this line but I think that it makes sense to just disable heap by default when building for TFM, right?
https://github.com/nrfconnect/sdk-nrf/blob/964fdb47b82449927fe7bcd9cb71daaa70d042f8/subsys/nrf_security/Kconfig#L92


Then we don't need to explicitly disable it for the non secure targets. Now if we have it enabled by default and we explicitly disable it in the samples it seems contradictive.

Reply:
Good point. Made the change